docs/guide/spot.md

```diff
@@ -0,0 +1,45 @@
+## Boston Dynamics' Atlas
+
+%robot spot images/robots/spot/model.thumbnail.png
+
+The "Spot", previously called "SpotMini", is is a four-legged dog-like robot developed by [Boston Dynamics](https://www.bostondynamics.com/spot).
+The nimble robot is 83 centimeters tall and is designed for a variety of search, inspection and delivery tasks.
+It can climbs stairs and traverses rough terrain with unprecedented ease, yet is small enough to use indoors.
+It is built to be a rugged (IP54 for dust and wet protection ) and customizable platform.
+Spot can go where wheeled robots cannot, while carrying payloads with endurance far beyond aerial drones.
+The maximal speed is 1.6 m/s with a runtime of 90 minutes and the batteries are swappable.
+Spot uses five stereo cameras (360 degrees vision) to avoid obstacles and people as it moves through dynamic work sites.
+
+### Spot PROTO
+
+Derived from [Robot](../reference/robot.md).
+
+```
+Spot {
+  SFVec3f     translation      0 -1.028 0.46
+  SFRotation  rotation         0 1 0 0
+  SFString    name             "Spot"
+  SFString    model            "Boston Dynamics - Spot"
+  SFString    controller       "moving_demo"
+  SFString    controllerArgs   ""
+  SFString    customData       ""
+  SFBool      supervisor       FALSE
+  SFBool      synchronization  TRUE
+  SFBool      selfCollision    FALSE
+  MFNode      extension        []
+}
+```
+
+> **File location**: "WEBOTS\_HOME/projects/robots/boston_dynamics/spot/protos/Spot.proto"
+
+#### Spot Field Summary
+
+- `extension`: Extends the robot with new nodes.

```

Comment:
You should probably mention where is located the extension (centre of the robot? in the head?).

Reply:
I forgot to remove it because at the moment I haven't found a 3D model of the extensions so I can't place the node correctly.

Reply:
Even if you don't but any extension it is good to keep it so that the user is free to add new sensors/actuators, but he just need to know where they will be added.